jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -249,15 +282,15 @@ cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/fullchain.pem
 cat /etc/letsencrypt/live/<CIRCLECI_SERVER_DOMAIN>/privkey.pem
 ```
```

Comment:
@circleci/server ??
These commands to retrieve certificates - do we need options other than using Let's Encrypt here? Also - these appear as part of the AWS section here but I think they should apply regardless of what platform people are on? If so I will move these below the GCP section to apply to all readers

Reply:
It's recommended that folks use the built-in ACME support to generate Let's Encrypt certificates. With that said, doing it manually is the only option if you're behind a proxy.

This isn't platform specific.